bindings/python/CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ set_target_properties(pyghex PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${python_mod_p
 # pyghex_obj because CMake.
 target_link_libraries(pyghex PRIVATE pybind11::module)
 target_link_libraries(pyghex PRIVATE ghex)
-ghex_link_to_oomph(pyghex)
+ghex_link_to_oomph(ghex)
```

Comment:
It seems to be some issue with transitive linkage.

Reply:
ok this seems to work!